bedrock/firefox/templates/firefox/os/index-2015.html

```diff
@@ -0,0 +1,14 @@
+{% extends "firefox/base-resp.html" %}
+
+{% add_lang_files "firefox/os/devices" %}
+
+{% block content %}
+{% set feed_links = firefox_os_feed_links(LANG) %}
```

Comment:
I forgot: yes, there two other pages that will use this, so it makes more sense as a helper.